### ISSUE TITLE
[colmap] Build error

### ISSUE THREAD

**Issue by QuanNguyen94** [microsoft/vcpkg#25609]
**Host Environment**

- OS: Window10
- Compiler: VS2019

**To Reproduce**
`./vcpkg install colmap[cuda,tests]:x64-windows`

**Failure logs**
```
Installing 1/1 colmap:x64-windows...
Building colmap[core,cuda,tests]:x64-windows...
-- Using cached colmap-colmap-29a1e3642a3b00734a52b21e597ea4d576485fe6.tar.gz.
-- Cleaning sources at F:/vcpkg/buildtrees/colmap/src/d576485fe6-7dd36362d1.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source F:/vcpkg/downloads/colmap-colmap-29a1e3642a3b00734a52b21e597ea4d576485fe6.tar.gz
-- Applying patch fix-dependencies.patch
-- Using source at F:/vcpkg/buildtrees/colmap/src/d576485fe6-7dd36362d1.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows-dbg
-- Configuring x64-windows-rel
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:129 (message):
    Command failed: F:/vcpkg/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: F:/vcpkg/buildtrees/colmap/x64-windows-dbg
    See logs for more information:
      F:\vcpkg\buildtrees\colmap\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/colmap/portfile.cmake:53 (vcpkg_cmake_install)
  scripts/ports.cmake:146 (include)
```

[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/9059871/install-x64-windows-dbg-out.log)

I think the bug is from the wrong directory path, but how to fix it, pls help!

**Comment by LilyWangLL** [comment]
Thanks for posting this issue. My local test blocked by `metis:x64-windows` installed failed, the reason is download failed. I will confirm this issue soon.

**Comment by DCNemesis** [comment]
I'm experiencing the same issue. I notice METIS is available on github. Is there a way to install metis from another source and skip it in the vcpkg install?

**Comment by LilyWangLL** [comment]
> I'm experiencing the same issue. I notice METIS is available on github. Is there a way to install metis from another source and skip it in the vcpkg install?

Could you please provide the related URL of METIS on github?

**Comment by DCNemesis** [comment]
https://github.com/KarypisLab/METIS

**Comment by orngeatom** [comment]
any updates here?

**Comment by LilyWangLL** [comment]
Now `colmap` block by a [Visual Studio bug](https://developercommunity.visualstudio.com/t/fatal-error-C1001:-Internal-compiler-err/10076899), the ticket submitted on [Developer Community](https://developercommunity.visualstudio.com/report?space=62&entry=problem) can effectively increase the priority of this bug being fixed.
Closed as duplicate of #25303.
